x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -259,6 +270,7 @@ async function installPackageFromRegistry({
   spaceId,
   force = false,
   ignoreConstraints = false,
+  neverIgnoreVerificationError = false,
```

Comment:
From an API point of view, force means "ignore version check and verification check" However internally sometimes we want to skip the version check but not the verification check. For example on install when we allow a user to install an outdated package if they dont have a leter version installed https://github.com/elastic/kibana/pull/136108/files#diff-cf9cec44de2ad0a6a3b74cca05e5308231d57d5c3e180ae4bc5114c2bf9af4ebR134

Reply:
The `neverIgnore` naming here seems a little confusing with the kind of double negative in place. Could this be a "positive" condition defaulting to true instead to make this more clear? Maybe like `surfaceVerificationErrors` or `forceVerificationErrors` or something along those lines? 

I understand the idea here is something like
- `force` by itself will suppress verification errors and allow installing outdated package version
- `force` with an additional flag should still allow for installing any package version, but should still surface verification errors